libsimba/auth/__init__.py

```diff
@@ -117,7 +117,7 @@ def cache_token(self, client_id: str, token: AuthToken) -> None:
             os.makedirs(token_dir, exist_ok=True)
             token_file = os.path.join(token_dir, "{}_token.json".format(client_id))
             with open(token_file, "w") as t1:
-                json_data = token.json(encoder=DateTimeEncoder())
```

Comment:
Can the dateTimeEncoder class be removed now?

Reply:
yep, removed!